packages/@webex/plugin-meetings/src/multistream/remoteMedia.ts

```diff
@@ -118,8 +118,10 @@ export class RemoteMedia extends EventsScope {
       fs = 240;
     } else if (height < 540) {
       fs = 920;
-    } else {
+    } else if (height < 900) {
```

Comment:
I'm thinking that with the threshold 900 I won't see 1080p in a meeting with just 1 other person on my 1080p monitor (because the UI around the video element takes up so many pixels), so maybe better to just put the threshold as 720 and we won't anyway request 1080p on more than 1 remote video, because we have a limit for the total of all remote video resolutions to be <= 1080p